transmonee_dashboard/src/transmonee_dashboard/pages/child_education.py

```diff
@@ -28,7 +28,7 @@
         ],
         "MAIN": {
             "name": "Out-of-School Children",
-            "geo": "Geographic area",
+            "geo": "REF_AREA",
```

Comment:
Have to rename all descriptions to codes, which is better for config, then will as descriptions back later